.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+#### I am submitting a -
+* [] **Feature Request**
+* [] **Bug Report**
+
+#### Describe your bug/feature -
+
+#### Current behaviour of the bug -
+
+#### Expected behaviour of the bug -
```

Comment:
"Expected behavior of the bug" looks odd to me. Just Current behaviour and expected behavior is enough

Reply:
@tachyons Ok will do so